spm_FIR_basis/batch.m

```diff
@@ -0,0 +1,132 @@
+matlabbatch{11}.spm.stats.fmri_spec.sess.regress = struct('name', {}, 'val', {});
+matlabbatch{11}.spm.stats.fmri_spec.sess.multi_reg = {''};
+matlabbatch{11}.spm.stats.fmri_spec.sess.hpf = 128;
+matlabbatch{11}.spm.stats.fmri_spec.fact = struct('name', {}, 'levels', {});
+matlabbatch{11}.spm.stats.fmri_spec.bases.fir.length = 20;
+matlabbatch{11}.spm.stats.fmri_spec.bases.fir.order = 10;
+matlabbatch{11}.spm.stats.fmri_spec.volt = 1;
+matlabbatch{11}.spm.stats.fmri_spec.global = 'None';
+matlabbatch{11}.spm.stats.fmri_spec.mthresh = 0.8;
+matlabbatch{11}.spm.stats.fmri_spec.mask = {''};
+matlabbatch{11}.spm.stats.fmri_spec.cvi = 'AR(1)';
+matlabbatch{12}.spm.stats.fmri_est.spmmat(1) = cfg_dep('fMRI model specification: SPM.mat File', substruct('.','val', '{}',{11}, '.','val', '{}',{1}, '.','val', '{}',{1}), substruct('.','spmmat'));
+matlabbatch{12}.spm.stats.fmri_est.write_residuals = 0;
+matlabbatch{12}.spm.stats.fmri_est.method.Classical = 1;
+matlabbatch{13}.spm.stats.con.spmmat(1) = cfg_dep('Model estimation: SPM.mat File', substruct('.','val', '{}',{12}, '.','val', '{}',{1}, '.','val', '{}',{1}), substruct('.','spmmat'));
+matlabbatch{13}.spm.stats.con.consess{1}.fcon.name = 'Tone Counting vs Baseline';
+matlabbatch{13}.spm.stats.con.consess{1}.fcon.weights = [1 0 0 0 0 0 0 0 0 0
+                                                         0 1 0 0 0 0 0 0 0 0
+                                                         0 0 1 0 0 0 0 0 0 0
+                                                         0 0 0 1 0 0 0 0 0 0
+                                                         0 0 0 0 1 0 0 0 0 0
+                                                         0 0 0 0 0 1 0 0 0 0
+                                                         0 0 0 0 0 0 1 0 0 0
+                                                         0 0 0 0 0 0 0 1 0 0
+                                                         0 0 0 0 0 0 0 0 1 0
+                                                         0 0 0 0 0 0 0 0 0 1];
+matlabbatch{13}.spm.stats.con.consess{1}.fcon.sessrep = 'none';
+matlabbatch{13}.spm.stats.con.delete = 0;
+matlabbatch{14}.spm.stats.results.spmmat(1) = cfg_dep('Contrast Manager: SPM.mat File', substruct('.','val', '{}',{13}, '.','val', '{}',{1}, '.','val', '{}',{1}), substruct('.','spmmat'));
+matlabbatch{14}.spm.stats.results.conspec.titlestr = '';
+matlabbatch{14}.spm.stats.results.conspec.contrasts = Inf;
+matlabbatch{14}.spm.stats.results.conspec.threshdesc = 'none';
+matlabbatch{14}.spm.stats.results.conspec.thresh = 0.001;
+matlabbatch{14}.spm.stats.results.conspec.extent = 0;
+matlabbatch{14}.spm.stats.results.conspec.conjunction = 1;
+matlabbatch{14}.spm.stats.results.conspec.mask.none = 1;
+matlabbatch{14}.spm.stats.results.units = 1;
+matlabbatch{14}.spm.stats.results.print = 'pdf';
+matlabbatch{14}.spm.stats.results.write.none = 1;
```

Comment:
Looking at the batch.m file the excursion set image is not saved, yet it is available in the spm_FIR folder. I am wondering which one is outdated (the excursion set image or the batch script)?